src/app/controllers/admin-forms.server.controller.js

```diff
@@ -715,5 +715,44 @@ function makeModule(connection) {
         },
       )
     },
+
+    /**
+     * Transfer a form to another user
+     * @param  {Object} req - Express request object
+     * @param  {Object} res - Express response object
+     */
+    transferOwner: async function (req, res) {
+      const newOwnerEmail = req.body.email
+
+      // Check if new owner's email is specified
+      if (_.isEmpty(newOwnerEmail)) {
+        const errMsg = 'Error: email should exists in req body'
+        logger.error(
+          `formId="${req.form._id}", ip=${getRequestIp(
+            req,
+          )}, message="${errMsg}"`,
+        )
+        return res.status(HttpStatus.BAD_REQUEST).send({ message: errMsg })
+      }
+
+      // Transfer owner and Save the form
+      try {
+        await req.form.transferOwner(req.session.user, newOwnerEmail)
```

Comment:
I would think its better if this method transfers ownership and also saves the form?

Reply:
@liangyuanruo didn't want to return database errors in that function so I removed it.

Reply:
got it